setup.py

```diff
@@ -90,7 +94,7 @@ def get_generator_flag():
 
 
 def get_compute_capabilities(
-    compute_capabilities={75, 80, 86, 89, 90}
+    compute_capabilities=set(map(int, COMPUTE_CAPABILITIES.split(",")))
```

Comment:
```python
>>> import os
>>> COMPUTE_CAPABILITIES = os.getenv("COMPUTE_CAPABILITIES", "75,80,86,87,89,90")
>>> set(map(int, COMPUTE_CAPABILITIES.split(",")))
{75, 80, 86, 87, 89, 90}
```
Should always create a set of unique integer numbers from `$COMPUTE_CAPABILITIES` (even if it contains doubles).